### DISCUSSION TITLE
let us save our config to ~/.config

### DISCUSSION THREAD

**Discussion by killertofus** [Chatterino/chatterino2#5555] [Ideas] [answered]
.

**Comment by pajlada** [comment]
Would a runtime parameter be useful? e.g. you'd start chatterino like this: `chatterino --base-config-dir-xd=~/config`?

**Comment by killertofus** [comment]
I would like it to be with `XDG_CONFIG_HOME` as most apps with configs, they have a option to keep it in ~/.config for easy access, as of now it keeps it in` $HOME/.local/share/chatterino` which is not ideal for keeping consistent with other  `XDG_CONFIG_HOME` apps

**Reply by pajlada** [reply, marked as answer]
I'm not interested in doing a path migration at this time

**Comment by Mm2PL** [comment]
Chatterino uses `XDG_DATA_HOME`, it's not a hardcoded `~/.local/share/chatterino`.

**Reply by killertofus** [reply]
didn't work for me unless I moved it to ~/.local/share/chatterino

**Reply by killertofus** [reply]
![image](https://github.com/user-attachments/assets/5220cac4-47b8-425f-99f0-0055327c1371)

**Reply by killertofus** [reply]
![image](https://github.com/user-attachments/assets/f5e0c981-35b6-4665-8019-e89457a64c2b)
![image](https://github.com/user-attachments/assets/e57edaee-8993-4321-b728-20e50f86f8bc)
![image](https://github.com/user-attachments/assets/89bea953-3a13-4e0b-ab42-be5d46acce85)
